BREAKING.md

```diff
@@ -210,6 +210,26 @@ It has been deprecated and is expected to be removed in the next major release.
 
 `IConnectableRuntime.deltaManager` in the same package is no longer used and deprecated as well.
 
+## 2.0.0-internal.3.3.0 Breaking changes
+
+-   [Ability to enable grouped batching](#Ability-to-enable-grouped-batching)
+
+### Ability to enable grouped batching
+
+The `IContainerRuntimeOptions.enableGroupedBatching` option has been added to the container runtime layer and is off by default. This option will group all batch messages
+under a new "grouped" message to be sent to the service. Upon receiving this new "grouped" message, the batch messages will be extracted and given
+the sequence number of the parent "grouped" message.
+
+Upon enabling this option, if any issues arise, use the `Fluid.ContainerRuntime.DisableGroupedBatching` feature flag to disable at runtime.
+
+This option will change a couple of expectations around message structure and runtime layer expectations. Only enable this option after testing
+and verifying that the following expectation changes won't have any effects:
```

Comment:
I think it has to have a message that this option could be enabled only after application ensures that 99.9% of its sessions contains these changes.